week2/project/ecommerce2/src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import ProductDetails from './components/ProductDetails';
+import NotFoundPage from './pages/NotFoundPage';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
```

Comment:
when you use Router you can use it instead of react strict mode
✏️need to modify✏️

Reply:
fixed